server/services/user.js

```diff
@@ -1,72 +1,63 @@
-import _ from 'lodash';
-// import debug from 'debug';
-// use old rxjs
 import { Observable } from 'rx';
+import _ from 'lodash';
 
-const publicUserProps = [
-  'id',
-  'name',
-  'username',
-  'bio',
-  'theme',
-  'picture',
-  'points',
-  'email',
-  'languageTag',
-
-  'isCheater',
-  'isGithubCool',
-
-  'isLocked',
-  'isAvailableForHire',
-  'isFrontEndCert',
-  'isBackEndCert',
-  'isDataVisCert',
-  'isFullStackCert',
-  'isRespWebDesignCert',
-  'isFrontEndLibsCert',
-  'isJsAlgoDataStructCert',
-  'isApisMicroservicesCert',
-  'isInfosecQaCert',
-
-  'githubURL',
-  'sendMonthlyEmail',
-  'sendNotificationEmail',
-  'sendQuincyEmail',
-
-  'currentChallengeId',
-  'challengeMap'
-];
-
-// const log = debug('fcc:services:user');
+import {
+  userPropsForSession,
+  normaliseUserFields,
+  getProgress
+} from '../utils/publicUserProps';
 
 export default function userServices() {
   return {
     name: 'user',
-    read: (req, resource, params, config, cb) => {
-      const { user } = req;
+    read: async function readUserService(
```

Comment:
Why is this an async func?

Reply:
That is left over from when I used `User.find$` to find a third party profile. It can be removed